common.h

```diff
@@ -48,10 +48,10 @@ typedef uint64_t cycles_t;
 #define TEXTDOMAIN "gbsplay"
 #define N_(x) x
 
-#define LN2 .69314718055994530941
-#define MAGIC 5.78135971352465960412
```

Comment:
For reference: MAGIC is log(55)/LN2 (https://github.com/mmitch/gbsplay/blame/78ee35cb0a5d60069fc7f4abc84abbebcda30d0a/gbsplay.c#L32)